src/main/java/mate/academy/intro/validation/FieldMatchValidator.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.intro.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.apache.commons.beanutils.BeanUtils;
+
+public class FieldMatchValidator implements ConstraintValidator<FieldMatch, Object> {
+    private String firstFieldName;
+    private String secondFieldName;
+
+    @Override
+    public void initialize(FieldMatch constraintAnnotation) {
+        this.firstFieldName = constraintAnnotation.first();
+        this.secondFieldName = constraintAnnotation.second();
+    }
+
+    @Override
+    public boolean isValid(Object value, ConstraintValidatorContext context) {
+        try {
+            String firstValue = BeanUtils.getProperty(value, firstFieldName);

```

Comment:
You can use Spring BeanWrapperImpl class
```suggestion
            String firstValue = new BeanWrapperImpl(value).getPropertyValue(this. firstFieldName);
```